internal/apis/kfd/v1alpha2/onpremises/create/preflight.go

```diff
@@ -0,0 +1,292 @@
+// Copyright (c) 2017-present SIGHUP s.r.l All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package create
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"path"
+	"path/filepath"
+
+	diffx "github.com/r3labs/diff/v3"
```

Comment:
we usually use `x` as a suffix for our internal "x" libraries. how about `r3diff`